server/src/main/java/io/druid/metadata/SQLMetadataStorageActionHandler.java

```diff
@@ -103,7 +109,19 @@ public Void withHandle(Handle handle) throws Exception
             {
               handle.createStatement(
                   String.format(
-                      "INSERT INTO %s (id, created_date, datasource, payload, active, status_payload) VALUES (:id, :created_date, :datasource, :payload, :active, :status_payload)",
+                      "INSERT INTO %s (%s_id, owner_id) VALUES (:id, :owner_id)",
+                      entryOwnerTable,
+                      entryTypeName
+                )
+              )
+                  .bind("id", id)
+                  .bind("owner_id", ownerId)
+                  .execute();
```

Comment:
This needs to be able to recover from failure. If the overlord dies between here and the next statement, will the table just continue to grow with stale entries in it?